app/services/brapi/v1/studies_queries.rb

```diff
@@ -0,0 +1,289 @@
+      plant_trials.place_name as "name",
+      countries.country_code as "countryCode",
+      countries.country_name as "countryName",
+      plant_trials.latitude as "latitute",
+      plant_trials.longitude as "longitude",
+      plant_trials.altitude as "altitude",
+      plant_trials.terrain as "terrain",
+      plant_trials.soil_type as "soil_type",
+      -- contacts
+      plant_trials.contact_person as "email",
+      plant_trials.entered_by_whom as "entered_by_whom_email",
+      -- additional info
+      plant_trials.plant_trial_description as "studyDescription",
+      plant_trials.data_provenance as "dataProvenance",
+      plant_trials.data_owned_by as "dataOwnedBy"    
+      
+    SQL
+    
+    # TODO: "studyType": "Trial" -> NOT PRESENT IN OUR DB
+    # TODO: "seasons": ["2007 Spring", "2008 Fall"] -> seasons themselves not present in our DB
+    # TODO: "startDate": "2007-06-01" -> NOT PRESENT IN OUR DB
+    # TODO: "endDate": "2008-12-31" -> NOT PRESENT IN OUR DB
+    # TODO: "active": true -> TO DEFINE ITS MEANING
+    
+    # TODO: "abbreviation": "IB" -> NOT PRESENT IN OUR DB
+    # TODO: "additionalInfo" : { "property1Name" : "property1Value"} -> TO DEFINE ITS MEANING
+    
+    # joins
+    joins_query = 
+    "FROM plant_trials
+    LEFT JOIN plant_scoring_units ON plant_trials.id = plant_scoring_units.plant_trial_id
+    LEFT JOIN plant_accessions ON plant_scoring_units.plant_accession_id = plant_accessions.id
+    LEFT JOIN countries ON plant_trials.country_id = countries.id
+    "
+    
+    total_query = select_query + joins_query + where_query
+    
+    json_wrapping_query = "SELECT row_to_json(row) from ("+total_query+") row"
+    result_object = execute_statement(json_wrapping_query, where_atts)
+    
+    return result_object
+  end
+
+
+  private
+
+  def get_select_distinct_clause(sortBy)
+    distinct_clause = ""
+    sort_by_field = get_sortby_field(sortBy) 
+    if sort_by_field == " plant_trials.id "
+      distinct_clause = "SELECT DISTINCT ON (plant_trials.id)"
+    else
+      distinct_clause = "SELECT DISTINCT ON (plant_trials.id,"+sort_by_field+")"
+    end 
+    return distinct_clause
+  end
+
+  def get_sortby_field(sortBy)
+    order_query = ""
+    case sortBy 
+    when "studyDbId"   
+      order_query += " plant_trials.id "
+    when "name"
+      order_query += " plant_trials.plant_trial_name "
+    when "trialDbId"
+      order_query += " plant_trials.id "
+    when "trialName"
+      order_query += " plant_trials.plant_trial_name "
+    when "seasons"
+      order_query += " plant_trials.trial_year "
+    when "locationDbId"
+      order_query += " countries.id "
+    when "locationName"
+      order_query += " countries.country_name "
+    when "programName"
+      order_query += " plant_trials.project_descriptor "
+    else
+      order_query += " plant_trials.id "
+    end
+    return order_query    
+  end
+
+  def get_posgresql_where_condition(field, value, condition_number )
```

Comment:
I included the mention to postgresql because I used a syntax which could be highly dependent on it (there, and in other methods)... so, do I remove that mention in all of them?